base/errorshow.jl

```diff
@@ -1086,6 +1092,28 @@ end
 
 Experimental.register_error_hint(methods_on_iterable, MethodError)
 
+# Display a hint in case the use tries to access non-member fields of container type datastructures
+function fielderror_hint_handler(io, exc)
+    @nospecialize
+    field = exc.field
+    type = exc.type
+    if type == :Dict
```

Comment:
I was expecting a type `Dict` but I was a bit surprised to see I actually get `Symbol`. I should familiarize with internals more. But if anyone sees this as anomaly let me know.

Reply:
Yes, that's seriously broken. I think the struct construction on the C side is buggy.